corehq/apps/userreports/sql.py

```diff
@@ -101,3 +104,33 @@ def _hash(domain, table_id):
 
     return 'config_report_{0}_{1}_{2}'.format(domain, table_id, _hash(domain, table_id))
 
+
+def get_expanded_columns(table_name, column_config):
+
+    session = Session()
+    connection = session.connection()
+    metadata = sqlalchemy.MetaData()
+    metadata.reflect(bind=connection)
```

Comment:
This is quite inefficient since it reflects every single table in the DB. Its probably better to use [get_indicator_table](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/sql.py#L54-54) to get the table object.